db/migrate/20220103182104_add_categories_to_purchases.rb

```diff
@@ -0,0 +1,10 @@
+class AddCategoriesToPurchases < ActiveRecord::Migration[6.1]
```

Comment:
What do you think about adding `_spent_` inside the column name? Based on the PR description I think it makes ti more clear what these columns are meant to describe.

Reply:
The column names are already pretty long. Considering these are attributes of a `Purchase`, I'm not sure what else "money" could mean in this context? :)

Reply:
I think you right to say that `Purchase` would be indicative of what this means. But as someone who doesn't have the full picture the addition of the word like `spent` helps define what this actually means. 

> The column names are already pretty long.

Agreed! However, I'am okay with columns being longer as long as the benefit outweighs the cost. In this case, I believe the word `_spent` in this would be helpful.

`amount_spent_on_diapers_cents`
`amount_spent_on_adult_incontinence_cents`
`amount_spent_on_other_cents`

How do these read? I guess it may be redundant but I usually optimize for clarity and not DRY.

What do you think?